src/User32.Desktop/User32+WNDCLASS.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="WNDCLASS"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
```

Comment:
The fields of this struct are not in their proper order, surely leading to bad interop.